Add April 2024 newsletter (#2127)
writethedocs/www: docs/blog/newsletter-april-2024.rst

> .. post:: April 04, 2024

Author: ```suggestion
:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: April 04, 2024
```

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: April 04, 2024
  :tags: newsletter

#########################################
Write the Docs Newsletter – April 2024
#########################################

Hail to you, documentarians! I am writing with birds singing outside my window to celebrate the coming of spring (or so I tell myself). I hope you are able to find some sounds that bring you hope for the future.

In the near future, the `Portland 2024 conference </conf/portland/2024/>`__ is starting on April 14th. I hope all of you who are able to attend come away with amazing ideas and connections. For the rest of us, Write the Docs Atlantic 2024 has now opened `ticket sales and grant applications </conf/atlantic/2024/news/tickets-grants/>`__. Plus the `Call for Proposals </conf/atlantic/2024/cfp/>`__ is still open until May 15th, so submit your proposals while you can.

> Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The most ever responses for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be in Portland this year, we're also going to be running a Writing Day session to improve the clarity, inclusivity, and relevance of the questions. 

Reviewer: Should it be `most-ever responses`?

Author: ```suggestion
Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The most-ever responses for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be in Portland this year, we're also going to be running a Writing Day session to improve the clarity, inclusivity, and relevance of the questions. 
```

.. post:: April 04, 2024
  :tags: newsletter

#########################################
Write the Docs Newsletter – April 2024
#########################################

Hail to you, documentarians! I am writing with birds singing outside my window to celebrate the coming of spring (or so I tell myself). I hope you are able to find some sounds that bring you hope for the future.

> In the near future, the `Portland 2024 conference </conf/portland/2024/>`__ is just around the corner, starting on April 14th. I hope all of you who are able to attend come away with amazing ideas and connections. For the rest of us, Write the Docs Atlantic 2024 has now opened `ticket sales and grant applications </conf/atlantic/2024/news/tickets-grants/>`__. Plus the `Call for Proposals </conf/atlantic/2024/cfp/>`__ is still open until May 15th, so submit your proposals while you can.

Reviewer 2: pick one of near 'future' / 'around the corner'?

Author: ```suggestion
In the near future, the `Portland 2024 conference </conf/portland/2024/>`__ is starting on April 14th. I hope all of you who are able to attend come away with amazing ideas and connections. For the rest of us, Write the Docs Atlantic 2024 has now opened `ticket sales and grant applications </conf/atlantic/2024/news/tickets-grants/>`__. Plus the `Call for Proposals </conf/atlantic/2024/cfp/>`__ is still open until May 15th, so submit your proposals while you can.
```

> Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The best ever response for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be attending Writing Day in Portland this year, we're going to be running a session to improve the clarity, inclusivity, and relevance of the questions. 

Reviewer 2: 'best ever response' sounds  bit like one response, not the most responses, but might just be me

Author: I think it was my thought as well, but more so since I changed it a bit.
```suggestion
Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The most ever responses for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be in Portland this year, we're also going to be running a Writing Day session to improve the clarity, inclusivity, and relevance of the questions. 
```

> Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The best ever response for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be attending Writing Day in Portland this year, we're going to be running a session to improve the clarity, inclusivity, and relevance of the questions. 

Reviewer 2: ```suggestion
Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The best ever response for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be attending Writing Day in Portland this year, we're also going to be running a Writing Day afternoon session to improve the clarity, inclusivity, and relevance of the questions. 
```

> Our articles this month cover the value of measuring docs by readability scores, whether AI is helping speed up docs creation right now, what questions to ask when hiring for a documentarian position, and whether and how to combine docs and code changes. I hope you enjoy and let us know what you're thinking about.

Reviewer 2: Very optional

```suggestion
Our articles this month cover the value of measuring docs by readability scores, whether AI is speeding up docs creation, what questions to ask when hiring a documentarian, and whether and how to combine docs and code changes. I hope you enjoy and let us know what you're thinking about.
```

These topics were relevant to a recent thread in the `#ai channel <https://writethedocs.slack.com/archives/C1NEAD7D4>`__. The question was whether, as some `public commenters have claimed <https://twitter.com/samjulien/status/1765866411404976282>`__, AI can already speed up the creation of product documentation.

Some respondents were adamant that it doesn't help them. Some because their company's security policies forbid its use. Others found it useful for generic content, where it can act like a focused web search but without any ads, but not for specialized questions about a product. Others said it wouldn't help them because writing the first draft is how they learn about the product.

Some of the people who didn't find it personally helpful thought it might help non-writers or non-native speakers. People who struggle getting words on the page might benefit from the ability of large-language models (LLMs) to put together sentences in a commonly understood order.

> Others were adamant that LLMs have really helped them speed up their documentation process. For security, it's important to choose a service that doesn't share prompts and responses with the LLM providers for training. 

Reviewer 2: Second use of adamant?

Author: It was supposed to be intentional (some are adamant that X, others are adamant that Y). 🤔

Reviewer 2: Hmmm, I think to me Adamant is a really specific level of belief, so I would not expect both sides to be equally specifically adamant. But I won't fight it, I'm not that adamant.

Author: From the conversation, it seemed like both sides were very strong in their beliefs. 🤷

----------------------------------
Questions To Ask as an Interviewer
----------------------------------

The WTD community offers valuable resources for job seekers, many captured in the `Guide to Hiring and Getting Hired </hiring-guide/>`__ and the `Getting Hired </topics/#getting-hired>`__ topics. This month in Slack, the focus shifted to the other side of the interview, as documentarians and managers listed their favorite questions to ask candidates. Here's a roundup of the questions they shared, along with any reasons they mentioned.

- How did you get into technical writing?

 - This is an easy starter question that gets candidates comfortable, gauges their enthusiasm for the job, and tells you about their career path.

- What do you do outside your regular role to advocate for your profession?

- Do you take the initiative on docs retros or create presentations for the team?

- Do you like to stretch beyond writing?

- What other profession would you like to try?

>  - This isn't a common interview question, so it shows how candidates respond to something they may not be prepared for.

Reviewer 2: This is a little too conditional for my tastes. Especially given the assumption.

> 
> - What other profession would you like to try?
> 
>  - This isn't a common interview question, so it shows how candidates respond to something they may not be prepared for.

Author: ```suggestion
```
